frontend/src/domain/mentoring/components/DetailHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from '@emotion/styled';
+
+import backButton from '../../../common/assets/images/backButton.svg';
+import Header from '../../../common/components/Header/Header';
+
+function DetailHeader() {
+  return (
+    <Header>
+      <StyledHeaderInner>
```

Comment:
StyledHeaderInner는 Header 내부에 존재해서 Inner를 붙인걸까?
그렇다면 Inner 대신 Wrapper는 어떨까??  🤔

Container는 최상위에서 쓰기로 해서 위치가 맞지 않는 것 같고, Header의 내부에 존재하니까 Wrapper를 붙여서 써도 될 것 같아! 😊
InnerWrapper도 좋을 것 같아 😊

```suggestion
      <StyledHeaderWrapper>
```

Reply:
안그래도 Wrapper를 써야하나 고민을 했었는데 Header 내부에 존재해서 Inner를 썼었어
다시보니 Wrapper로 통일하는 편이 더 좋은 것 같아
좋은 의견 땡큐 ☺️